pages/casestudies/[id].js

```diff
@@ -17,7 +17,7 @@ const renderContent = (content, allComponents, level) => {
   return content.map((item) => {
     return (
       <div
-        className="mt-10"
+        className={`mt-10 mx-auto w-11/12`}
```

Comment:
Rewrite this in conventional tailwind syntax as it was written previously

Reply:
you want to remove width as well? or just remove {} as there is no condition checking anymore?

Reply:
Just the braces